ocs_ci/ocs/warp.py

```diff
@@ -1,20 +1,24 @@
+import concurrent.futures as futures
+from io import StringIO
 import logging
 import os
+from tempfile import mkdtemp
+import threading
 import time
 
-import concurrent.futures as futures
+import pandas as pd
+
+from ocs_ci.framework import config
 from ocs_ci.helpers import helpers
-from ocs_ci.utility import templating
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed, UnexpectedBehaviour
 from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import pod
 from ocs_ci.ocs.resources.ocs import OCS
 from ocs_ci.ocs.resources.pod import Pod, get_pods_having_label
-from ocs_ci.ocs import constants
-from ocs_ci.framework import config
-from ocs_ci.ocs.resources import pod
-from tempfile import mkdtemp
-from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.utility import templating
 
-log = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
```

Comment:
I am not seeing any difference in `log` and `logger` variable.
Do we really need to change it to `logger`?